Automate/Speckle.Automate.Sdk/Schema/AutomationRunData.cs

```diff
@@ -1,8 +1,7 @@
-# nullable enable
 namespace Speckle.Automate.Sdk.Schema;
 
 ///<summary>
-///Values of the project, model and automation that triggere this function run.
+///Values of the project, model and automation that trigger this function run.
```

Comment:
I think the right tense would be `triggered` in this case

Reply:
resolved